install/include/hipSYCL/sycl/libkernel/marray.hpp

```diff
@@ -0,0 +1,518 @@
+/*
```

Comment:
There seems something wrong here. This adds code to an `marray.hpp` in an `install` directory rather than modifying  the actual existing header. Have you by accident added the wrong file?

Reply:
Whoops, not sure how that happened. I put the changes in the correct file.